### ISSUE TITLE
Renderer default listers should have a low priority

### ISSUE THREAD

**Issue by pjasiun** [ckeditor/ckeditor5-engine#644]
We agreed that the proper way of sync-ing UI features with the engine is listening on the `viewDocument.render`. UI features may want to change something in the view on rendering (for instance add class to the element). Such changes should be done before `renderer()` is called. This is why this code: https://github.com/ckeditor/ckeditor5-engine/blob/98e3d4fb230c8bcf6d66fc816fe8704cf6263f2a/src/view/document.js#L115-L119 should have a low priority.

**Comment by pjasiun** [comment]
On the other hand, UI feature may expect that DOM is already updated. Then `renderer` should be called with `high` priority. Anyway adding it with the default priority it the worst scenario, because then default listeners have the same priority what does not define a clear order for them.

Also, it should be clearly mentioned in the documentation how to add action before and after rendering.

**Comment by pjasiun** [comment]
Fixed by https://github.com/ckeditor/ckeditor5-engine/issues/1210.
